spec/src/consensus.rs

```diff
@@ -97,7 +97,7 @@ impl Default for Consensus {
             id: "main".to_owned(),
             max_uncles_age: MAX_UNCLE_AGE,
             max_uncles_num: MAX_UNCLE_NUM,
-            epoch_reward: capacity_bytes!(5_000_000),
+            epoch_reward: capacity_bytes!(500_000),
```

Comment:
The config in `testnet.toml` is 5,000,000 bytes, but the default value here is 500,000 bytes. Is it by design that the default value is for mainnet?

Reply:
`DEFAULT_SECONDARY_EPOCH_REWARD` is the same, the default value is 1/10 of the one in `testnet.toml`.

Reply:
And we'd better keep consistent here, all the other fields use constants, only this one uses hard-coded number.